src/stories/contextMenuAndSelection/ContextMenuAndSelection.stories.tsx

```diff
@@ -0,0 +1,113 @@
+                                    run: (param) => {
+                                        param.closeContextMenu();
+                                        const selectedTask = tasks.find((task) => task.key == param.selection[0]);
+                                        // needed setTimeout for menu to have time to close
+                                        setTimeout(()=> {
+                                            let newTitle = prompt("Task new title:", selectedTask.title);
+                                            newTitle && setTasks(tasks.map((task) => task == selectedTask ? { ...task, title: newTitle } : task));
+                                        }, 10); 
+                                    }
+                                },
+                                {
+                                    icon: "trash",
+                                    label: "Delete",
+                                    isVisible: (param) => { return param.selection.length == 1; },
+                                    renderInMenu: (param) => {
+                                        return <Menu.Item onClick={() => { 
+                                                        setTasks(tasks.filter(task => !param.selection.includes(task.key)));
+                                                        param.closeContextMenu();}}>
+                                                    {/** This is a trivial example for customizing the content of an action renderer 
+                                                     * but this content can be replaced according to application needs with a more complex one: 
+                                                     * e.g. maybe containing a color picker (for an action that changes the color of a segment)*/}        
+                                                    <span style={{color: "red"}}>Delete</span> 
+                                                    <Icon name="trash" color='red'/>      
+                                                </Menu.Item>
+                                    }
+                                }
+                            ];
+                            // We can filter the actions that will be displayed directly here in the actions provider    
+                            if (contextMenuShowParam.actionParam.selection.length == 0 && contextMenuShowParam.actionParam.row < someHumanResources.length) {
+                                actions.push({
+                                    icon: "plus",
+                                    label: (param) => {
+                                        return "Add task for " + someHumanResources[param.row].title
+                                    },
+                                    run: (param) => {
+                                        setTasks([...tasks, { key: tasks.length, row: param.row, title: 'NEW TASK', start: d('2018-09-20 1:00'), end: d('2018-09-20 3:00') }]);
```

Comment:
Ar fi bine sa avem in param info si pentru axa x. Astfel, vom putea sa cream segmentul incepand cu pozitia mouse-ului

Reply:
ok. am adaugat in parametru de rulare si 'time' (timpul la care s-a click-at pentru a se deschide meniul)